docs/usage/security-and-permissions.md

```diff
@@ -1,6 +1,33 @@
 # Security and Permissions
 
-## Global Permissions
+This page talks about our security stance, and explains what permissions are needed for the different ways you can run Renovate.
+
+## Security Stance
+
+Renovate is open source software, and comes with no guarantees or warranties of any kind.
+That said, we will try to fix security problems in a reasonable timeframe if possible.
+
+If you need commercial level support then use INSERT OUR COMMERCIAL OFFERING HERE.
+
+### No certifications
+
+Renovate does **not** have ISO 27001 or SOC2 certification.
```

Comment:
I'm adding a certification section because it was asked about by a potential customer.